libs/constants/src/types/models/models.ts

```diff
@@ -523,6 +523,58 @@ export interface RunSessionTimestamp {
   createdAt: DateString;
 }
 
+export interface RunSplits {
+  trackStats: RunStats;
+  segments: RunSegment[];
+}
+
+export interface RunSegment {
+  subsegments: RunSubsegment[];
+
+  // Contains an entry for every subsegment the player has reached so far
```

Comment:
This comment should be in front of `subsegments` (the original C++ has the same error lol)